library-backend/library-api/src/main/java/spring/learning/video/three/DrawingApp.java

```diff
@@ -0,0 +1,15 @@
+package spring.learning.video.three;
+
+import org.springframework.beans.factory.BeanFactory;
+import org.springframework.beans.factory.xml.XmlBeanFactory;
+import org.springframework.core.io.FileSystemResource;
+
+public class DrawingApp {
+
+    public static void main(String[] args) {
+//        Triangle triangle = new Triangle();
+        BeanFactory factory = new XmlBeanFactory(new FileSystemResource("D:\\library\\library\\library-backend\\library-api\\src\\main\\resources\\spring.xml"));
```

Comment:
Never use and commit `absolute paths`. Other colleagues might not have the same folder structure as you or the same operating system.

Check [here](https://stackoverflow.com/questions/14022839/how-do-i-load-a-resource-and-use-its-contents-as-a-string-in-spring) how to reference the file in the resource folder.

Reply:
This might work even better. The previous example is loading the content of the resource file.

```java
    ApplicationContext springAppContext = new ClassPathXmlApplicationContext("AllSpringBeanSettings.xml");
```